app/api/routers/query.py

```diff
@@ -25,3 +26,11 @@ async def get_query(query: QueryModel = Depends(QueryModel)):
     )
 
     return response
+
+
+@router.get("/attributes/{attribute_URI}")
```

Comment:
```suggestion
@router.get("/attributes/{dataElementURI}")
```
I don't think we're fully settled yet on what we want to call these things, but attribute is probably not the right word. Either "variable" or "common data element" or "data element" makes more sense